src/treeffuser/baselines/nnffuser.py

```diff
@@ -10,23 +10,27 @@
 
 class NNffuser(BaseTabularDiffusion):
     """
-    A standard diffusion model with a neural network score model.
+    A standard diffusion model with a neural network score model. A CARD-like
+    architecture is also available.
     """
 
     def __init__(
         self,
         n_repeats: int = 10,
-        n_epochs: int = 100,
+        eval_freq: int = 100,
+        max_evals: int = 1000,
         early_stopping_rounds: int = 10,
         eval_percent: float = 0.1,
         use_separate_models: bool = True,
         learning_rate: float = 0.01,
         n_layers: int = 2,
         hidden_size: int = 10,
         batch_size: int = 32,
+        decay: float = 0.999,
         weight_decay: float = 0.0,
         n_jobs: int = -1,
         sde_name: str = "vesde",
+        card_like: bool = False,
         sde_initialize_from_data: bool = False,
         sde_hyperparam_min: Optional[float | Literal["default"]] = None,
         sde_hyperparam_max: Optional[float | Literal["default"]] = None,
```

Comment:
btw I think this typing should be updated to use Union since we want to support python 3.9

Reply:
Ok will change!